xml/System/Type.xml

```diff
@@ -3548,7 +3548,7 @@ If the current <xref:System.Type> represents a generic type parameter, the <xref
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Type.GetEvents%2A> method does not return events in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which events are returned, because that order varies.  
+ In .NET 6 and below versions the <xref:System.Type.GetEvents%2A> method does not return events in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which events are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```

Comment:
```suggestion
In .NET 6 and earlier versions, the <xref:System.Type.GetEvents%2A> method does not return events in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which events are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```